_posts/2020-05-29-cockpit-common-criteria.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Aligning Cockpit with Common Criteria
+author: marusak
+date: 2020-05-29
+category: announcement
+tags: cockpit common-criteria
+slug: cockpit-common-criteria
+comments: true
+---
+
+In the last few releases new features were delivered to make Cockpit correctly aligned to Common Criteria and thus making it possible to undergo the certification process in the near future.
```

Comment:
"make Cockpit meet the Common Criteria". This could link to https://www.commoncriteriaportal.org/